scylla/tests/integration/session.rs

```diff
@@ -831,6 +831,39 @@ async fn test_get_tracing_info(session: &Session, ks: String) {
     let tracing_info: TracingInfo = session.get_tracing_info(&tracing_id).await.unwrap();
     assert!(!tracing_info.events.is_empty());
     assert!(!tracing_info.nodes().is_empty());
+
+    // Check if the request type matches
+    assert_eq!(tracing_info.request.unwrap(), "Execute CQL3 query");
+
+    // Verify duration is positive when Scylla is used
+    if session
+        .get_cluster_state()
+        .get_nodes_info()
+        .first()
+        .unwrap()
+        .sharder()
+        .is_some()
+    {
+        assert!(tracing_info.duration.unwrap() > 0);
+    }
+    // Verify started_at timestamp is present
+    assert!(tracing_info.started_at.unwrap().0 > 0);
+
+    // Check parameters
+    assert!(tracing_info
+        .parameters
+        .clone()
+        .unwrap()
+        .contains_key("consistency_level"));
+    assert!(tracing_info.parameters.unwrap().contains_key("query"));
```

Comment:
:wrench: Instead of `clone()`ing an `Option<T>` in order to `unwrap()` it, it's better to use `as_ref()` to `unwrap()` on a borrowed version of the `Option`. See:

```suggestion
    // Check parameters
    assert!(tracing_info
        .parameters
        .as_ref()
        .unwrap()
        .contains_key("consistency_level"));
    assert!(tracing_info.parameters.as_ref().unwrap().contains_key("query"));
```

Reply:
adjusted